src/vstart.sh

```diff
@@ -542,6 +544,13 @@ start_mon() {
 			--cap mgr 'allow *' \
 			"$keyring_fn"
 
+		prun $SUDO "$CEPH_BIN/ceph-authtool" --gen-key --name=client.rgw --set-uid=0 \
+		    --cap mon 'allow *' \
+		    --cap osd 'allow *' \
+		    --cap mds 'allow *' \
+		    --cap mgr 'allow *' \
+		    "$keyring_fn"
```

Comment:
@theanalyst If we do not supply --name then we use "client.admin", so it will still work.
You are right, if we want to use "client.admin", then both [client.rgw] in ceph.conf and creating keyring for client.rgw are unnecessary.
But I followed existing logic of having separate entries for radosgw.